benchmark/single-source/DataBenchmarks.swift

```diff
@@ -169,16 +169,29 @@ public let DataBenchmarks = [
     legacyFactor: 50),
 
   BenchmarkInfo(name: "StringToDataEmpty",
-    runFunction: { data($0*200, from: emptyString) }, tags: d,
+    runFunction: { dataFromUTF8View($0*200, from: emptyString) }, tags: d,
     legacyFactor: 50),
   BenchmarkInfo(name: "StringToDataSmall",
-    runFunction: { data($0*200, from: smallString) }, tags: d,
+    runFunction: { dataFromUTF8View($0*200, from: smallString) }, tags: d,
     legacyFactor: 50),
   BenchmarkInfo(name: "StringToDataMedium",
-    runFunction: { data($0*200, from: mediumString) }, tags: d,
+    runFunction: { dataFromUTF8View($0*200, from: mediumString) }, tags: d,
     legacyFactor: 50),
   BenchmarkInfo(name: "StringToDataLargeUnicode",
-    runFunction: { data($0*200, from: largeUnicodeString) }, tags: d,
+    runFunction: { dataFromUTF8View($0*200, from: largeUnicodeString) }, tags: d,
+    legacyFactor: 50),
+
+  BenchmarkInfo(name: "String.data.Empty",
+    runFunction: { dataUsingUTF8Encoding($0*200, from: emptyString) }, tags: d,
+    legacyFactor: 50),
```

Comment:
These new benchmarks shouldn't use `legacyFactor`. That's only for getting old benchmarks to run under 1000 μs but report the old inflated runtimes…

As soon as we'll do the renaming of the tests, the runtimes multiplied with legacy factor will be reported with the old names and the real runtimes will be reported under new naming convention (`StringToDataMedium` will become `Data.init.String.Medium`). This way we'll maintain the continuity of long term performance tracking from previous releases until we transition to the fully cleaned up benchmark suite.
```suggestion
  ),
```